dsgrid/config/dataset_config.py

```diff
@@ -84,6 +84,7 @@ class InputDatasetType(DSGEnum):
     SECTOR_MODEL = "sector_model"
     HISTORICAL = "historical"
     BENCHMARK = "benchmark"
+    GROWTH_RATE = "growth_rate"
```

Comment:
I think GROWTH_RATE is distinct from InputDatasetType, that is, you could provide a growth_rate dataset based on a sector model, historical data, or benchmarke data.

How about a new enum like:

```
class DatasetQualifiers(DSGEnum):
    GROWTH_RATE = "growth_rate"
```

I imagine we may eventually want to add others. Right now "normalized" comes to mind.